products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,157 @@
-# Global Documentation Search
+# Documentation Search MVP
 
-Author: Bill Fienberg
-Last Updated: 2020-01-23
-Reviewers: TBD
+**Author(s):** Bill Fienberg <bill.fienberg@oddball.io>
 
-## Objective
+**Last Updated:** 2020-01-29
 
-## Background
+**Status:** **Draft**
+
+**Approvers:** Andrew G., Megan K., Rian F., DevOps
 
 ## Overview
 
-## Detailed Design
+### Objective
+
+Run our own instance of [Algolia's DocSearch](https://community.algolia.com/docsearch/what-is-docsearch.html) on a custom landing page.
+
+### Background
+
+The documentation we have is spread across many different locations (public repos, private repos, GitHub pages, handbooks, etc). There currently isn't a way for VFS/VSP users to search multiple documentation sources with a single query.
+
+### High Level Design
+
+We are adding a public landing page that will contain a search input where users can search multiple sources of documentation with one query.
+
+## Specifics
+
+### Detailed Design
+
+We are adding a public landing page that will contain the [DocSearch frontend library](https://github.com/algolia/docsearch), and running our own instance of the [`docsearch-scraper`](https://github.com/algolia/docsearch-scraper) to extract content from our documentation sources and push it to an Algolia index.
+
+A successful implementation will have these components.
+
+1. Page with Input Textbox - We need a public page with a search input where users can input their search term(s).
+1. Index/Database - We need a place to store the records that will be returned to the user.
```

Comment:
I thought the Index/Database is hosted by algolia. Am I missing something?

Reply:
You are correct. If we sign up for Algolia, then Algolia would host the index.